### PR TITLE
Feature/eval hyper tuning

### PR DESCRIPTION
**Pull request by beckstev**
Added script to analyze the hyperparamter tuning. 

__How to test__
`python eval_hyper_tuning.py <path to model>`

Please be aware that your model need a `hyper_param_tuning` directory which will be generated automatically during the hyper tuning. I can hand you a test directory tomorrow.

### CODE REVIEW

**Review by FeGeyer — CHANGES_REQUESTED** [review]
Good job, only switching from learning_rate to use_rgb is missing

**Comment by beckstev** [comment]
@FritzGeise 
Please find your requested changes in my latest commits.

File: scripts/hyper_tuning.py [hunk 1]
```diff
@@ -40,28 +40,24 @@
     if args.early_stopping_delta:
         early_stopping_delta = args.early_stopping_delta
 
-    if args.use_rgb:
```
Comment (FeGeyer) [inline scripts/hyper_tuning.py]: Brauchen wir das in der `training.py` eigentlich noch? Das wird meiner Ansicht nach nirgendwo verwertet...

**Review by FeGeyer — APPROVED** [review]
Hab die Ordnerbezeichung noch geändert und die learning rate wieder auf den Standard Wert gesetzt, der Rest war gut.
